cmd/template/cluster/flag.go

```diff
@@ -122,26 +120,25 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&f.AWS.EKS, flagAWSEKS, false, "Enable AWSEKS. Only available for AWS Release v20.0.0 (CAPA)")
 
 	// OpenStack only.
-	cmd.Flags().StringVar(&f.OpenStack.Cloud, flagOpenStackCloud, "openstack", "Name of cloud (OpenStack only).")
-	cmd.Flags().StringVar(&f.OpenStack.CloudConfig, flagOpenStackCloudConfig, "cloud-config", "Name of cloud config (OpenStack only).")
+	cmd.Flags().StringVar(&f.OpenStack.Cloud, flagOpenStackCloud, "", "Name of cloud (OpenStack only).")
```

Comment:
I am still a bit uncertain if maintaining all these flags as well as the config file input makes sense. It just seems like a lot of flags to me.

Reply:
I agree. After using both methods I'm finding the flags more usable. They are documented and discoverable. It avoids extra steps of editing a values file and then running a command. It restricts the interface (can't pass arbitrary values like you can with `values.yaml`) and can be validated.